test/math.jl

```diff
@@ -363,6 +370,12 @@ end
             @test cosd(convert(T,-90))::fT === zero(fT)
             @test cosd(convert(T,-270))::fT === zero(fT)
         end
+        @testset "sincosd" begin
+            @test sincosd(convert(T,  0))::fTsc === ( sin( convert(fT, 0)),     cos( convert(fT, 0))     )
+            @test sincosd(convert(T, 90))::fTsc === ( sin( convert(fT, pi/2)),  cos( convert(fT, pi/2))  )
```

Comment:
would be better to test against `(sind(...), cosd(...))`. Also check 30 and 60 as well (since `sind(30) == 0.5`).

Reply:
Hi @simonbyrne ,

After some tests and given the suggestions of @StefanKarpinski and @KristofferC to make `sincosd` gives the same answer of `sind` and `cosd` instead of `sincos(deg2rad(x))`, I changed the tests and the implementation itself. 

The new PR is #30134. In the tests, I am actually comparing against `zero(fT)` and `one(fT)`.